lib/src/application/get_servieces/get_services_state.dart

```diff
@@ -12,7 +11,17 @@ part 'get_services_state.freezed.dart';
 ///
 @freezed
 abstract class ServiceState with _$ServiceState {
```

Comment:
Dont use abstract class for State or Event while using factory constructors. Just use normal class

Reply:
finish